setuptools/tests/test_build_meta.py

```diff
@@ -44,6 +46,15 @@ def __init__(self, *args, **kwargs):
 
     def __call__(self, name, *args, **kw):
         """Handles aribrary function invocations on the build backend."""
+        with suppress(ValueError):
```

Comment:
I want to disentangle this workaround from the specifics of this test, which I've done in f767b4f59. What's interesting is that the tests pass just fine for me with or without the workaround.